contracts/bond/BondFactory.sol

```diff
@@ -19,7 +20,8 @@ contract BondFactory is
     BondCreator,
     CollateralWhitelist,
     UUPSUpgradeable,
-    Version
+    Version,
+    PausableUpgradeable
```

Comment:
nit: alphabetic ordering of imports.

Unless the structuring of data in memory is an issue, let's aim for an ordering. The choices I see are either alphabetical or by character length 😉

Reply:
Was actually wondering this as I was typing it - should have picked up that it was alphabetical (which sounds reasonable to me)